api/c/indigo/src/indigo_loaders.h

```diff
@@ -136,10 +136,12 @@ class IndigoRdfLoader : public IndigoObject
     std::unique_ptr<Scanner> _own_scanner;
 };
 
+using namespace rapidjson;
```

Comment:
Please avoid `using namespace ..` in header files.

Reply:
Fixed